qiskit/pulse/library/parametric_pulses.py

```diff
@@ -135,14 +136,15 @@ def draw(self, dt: float = 1,
             interp_method: A function for interpolation
             scale: Relative visual scaling of waveform amplitudes
             interactive: When set true show the circuit in a new window
+            draw_title: Add a title to the plot when set "True"
                 (this depends on the matplotlib backend being used supporting this)
```

Comment:
I think these lines should be swapped.

Reply:
I thought it's for all parameters. Could you suggest change please?

Reply:
Sure, see below.